### PR TITLE
Dashboard designer pages UI fixes

### PR DESCRIPTION
**Pull request by Minoli**

### CODE REVIEW

File: components/dashboards-web-component/src/designer/DashboardDesigner.css [hunk 1]
```diff
@@ -92,4 +92,8 @@
 
 .publishers-list {
     margin-top: 10px;
+}
+
+.home-page-checkbox {
+    padding: 30px 0px;
 }
```
Comment (jeradrutnam) [inline components/dashboards-web-component/src/designer/DashboardDesigner.css]: Newline at the end of file is missing. Please add a new line on end of the files.

File: components/dashboards-web-component/src/designer/components/PageEntry.jsx [hunk 2]
```diff
@@ -63,10 +70,10 @@ export default class PageEntry extends Component {
                                    this.dirty = true;
                                }}/>
                     <Checkbox label={<FormattedMessage id="make.homepage" defaultMessage="Make as Home Page"/>} checked={this.state.page.landingPage}
-                              disabled={this.state.page.landingPage} onCheck={this.makeAsHomePage.bind(this)}/>
+                              disabled={this.state.page.landingPage} onCheck={this.makeAsHomePage.bind(this)} className="home-page-checkbox"/>
```
Comment (jeradrutnam) [inline components/dashboards-web-component/src/designer/components/PageEntry.jsx]: Limit character length to 120

File: components/dashboards-web-component/src/designer/components/WidgetConfigurationPanel.jsx [hunk 3]
```diff
@@ -102,6 +108,7 @@ class WidgetConfigurationPanel extends React.Component {
 
     render() {
         return (<Drawer open={this.props.open} openSecondary={true}
+                        containerStyle={styles.widgetDrawer}
```
Comment (jeradrutnam) [inline components/dashboards-web-component/src/designer/components/WidgetConfigurationPanel.jsx]: Try use consistent formatting everywhere

File: components/dashboards-web-component/src/designer/components/PageEntry.jsx (hunk 2 above)
Comment (Minoli) [reply components/dashboards-web-component/src/designer/components/PageEntry.jsx]: Outdated, file not found in master

File: components/dashboards-web-component/src/designer/DashboardDesigner.css (hunk 1 above)
Comment (Minoli) [reply components/dashboards-web-component/src/designer/DashboardDesigner.css]: Outdated, file not found in master

**Pull request merged**